### ISSUE TITLE
Send multiple Headers with request

### ISSUE THREAD

**Issue by rob4629** [vesln/hippie#46]
Would it be possible to pass in multiple `header` args when making an API request? I have different APIs which require different headers... and it would be great to be able to pass in an object containing the headers (so I can DRY up the code a little). e.g:

```js
const swagger_headers = {
    'Content-Type': 'application/json',
    'Header2': 'SomeValue',
    'Header3': 'AnotherValue'
      }

hippie()
    .base(baseApi)
    .auth(username, password)
    .headers(swagger_headers)
    .get('/v1/endpoint')
    .expectStatus(200)
```

As it currently stands, I have to set each `header` individually:

```js
hippie()
    .base(baseApi)
    .auth(username, password)
    .header('Content-Type', 'application/json')
    .header('Header2', 'SomeValue')
    .header('Header3', 'AnotherValue')
    .get('/v1/endpoint')
    .expectStatus(200)
```

**Comment by rob4629** [comment]
I've opened a [PR](https://github.com/vesln/hippie/pull/49), with a new `headers` option

**Comment by rob4629** [comment]
Can be closed, due to #49.
